torch/CMakeLists.txt

```
@@ -265,7 +265,7 @@ add_custom_command(
     OUTPUT
     "${TORCH_SRC_DIR}/utils/data/datapipes/datapipe.pyi"
     COMMAND
-    ${CMAKE_COMMAND} -E env PYTHONPATH="${TORCH_ROOT}"
+    ${CMAKE_COMMAND} -E env PYTHONPATH="${TORCH_ROOT}:$ENV{PYTHONPATH}"
```

I'm not very sure that `:` is the right `pathsep` on Windows.

Reference:

- https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-E_env-modify
- https://cmake.org/cmake/help/latest/prop_test/ENVIRONMENT_MODIFICATION.html#prop_test:ENVIRONMENT_MODIFICATION

```suggestion
    ${CMAKE_COMMAND} -E env --modify PYTHONPATH=path_list_prepend:"${TORCH_ROOT}" --
```

---

Thanks, I knew there had to be a better solution!